app/src/main/res/values-te-rIN/strings.xml

```diff
@@ -0,0 +1,265 @@
+    <string name="set_up">ఏర్పాటు</string>
+    <string name="device_help">మీరు మీ స్మార్ట్ స్పీకర్తో ఉపయోగించాలనుకుంటున్న Wi-Fi నెట్వర్క్ని ఎంచుకోండి.</string>
+    <string name="connecting_device">మీ పరికరానికి కనెక్ట్ చేస్తోంది</string>
+    <string name="connection_success">విజయవంతంగా కనెక్ట్ చేయబడింది</string>
+    <string name="choose_wifi">Wi-Fi ని కనెక్ట్ చేయండి</string>
+    <string name="wifi_access">దయచేసి Wi-Fi ప్రాప్యతను అందించండి</string>
+    <string name="wifi_access_message">అందుబాటులో ఉన్న నెట్వర్క్ల కోసం శోధించడానికి Wi-Fi ని ఆన్ చేయండి</string>
+    <string name="enter_password">పాస్ వర్డ్ ను ఎంటర్ చెయ్యండి</string>
+    <string name="thanks_wifi">మీ WiFi ఆధారాలను నమోదు చేసినందుకు ధన్యవాదాలు</string>
+    <string name="enter_password_mail">ఇది చివరి దశ, స్పీకర్కు విజయవంతంగా కనెక్ట్ చేయడానికి మీ SUSI.AI ఖాతా పాస్వర్డ్ని నమోదు చేయండి</string>
+    <string name="button_post">పోస్ట్</string>
+    <string name="desc_post_feedback">వివరణ పోస్ట్ ఫీడ్బ్యాక్</string>
+    <string name="example_feedback">ఉదాహరణ వ్యాఖ్య</string>
+    <string name="feedback_section">అభిప్రాయం</string>
+    <string name="hint_feedback">వ్యాఖ్యానించే సామర్ధ్యం</string>
+    <string name="placeholder_date">25 జూన్ 2018</string>
+    <string name="post_feedback_for_anonymous_user">వ్యాఖ్యలను పోస్ట్ చెయ్యడానికి సైన్ ఇన్ చేయండి</string>
+    <string name="see_all_reviews">అన్ని సమీక్షలను చూడండి</string>
+    <string name="reviews">సమీక్షలు</string>
+    <string name="toast_empty_feedback">దయచేసి పోస్ట్కు వ్యాఖ్యను నమోదు చేయండి</string>
+    <string name="toast_feedback_updated">సామర్థ్య వ్యాఖ్య నవీకరించబడింది</string>
+    <string name="metric_staff_picks">ఉద్యోగుల ఎంపిక</string>
+    <string name="metric_rating">SUSI, మీ అత్యధిక నాణ్యత సామర్థ్యం ఏమిటి?</string>
+    <string name="metric_usage">సూసీ, మీ అత్యంత ప్రతిభావంతులైన ప్రతిభ ఏమిటి?</string>
+    <string name="metric_latest">SUSI, ఇటీవల అప్గ్రేడ్ చేయబడిన నైపుణ్యం ఏమిటి? మీ అత్యంత ప్రతిభావంతులైన నైపుణ్యాలు ఏమిటి?</string>
+    <string name="metric_feedback">SUSI, చాలా అభిప్రాయాలతో నైపుణ్యం ఏమిటి?</string>
+    <string name="metric_newest">యుఎస్ఐ, కొత్త టాలెంట్ ఏమిటి?</string>
+    <string name="metrics_top_games">SUSI, మీ ఉత్తమ ఆట ఏమిటి?</string>
+    <string name="message_no_skills_found">సందేశం ఏ నైపుణ్యం కనుగొనబడలేదు</string>
+    <string name="report_skill">తగనిదిగా ఫ్లాగ్ చేయండి</string>
+    <string name="report_send">నివేదిక పంపండి</string>
+    <string name="report_error">దయచేసి సంభావ్యతను నివేదించండి మరియు దయచేసి మళ్లీ ప్రయత్నించండి</string>
+    <string name="report_send_success">సామర్ధ్యం విజయవంతంగా ప్రకటించబడింది</string>
+    <string name="reported_already">సామర్థ్యం ఇప్పటికే ప్రకటించబడింది</string>
+    <string name="placeholder_email">abc@example.com</string>
+    <string name="placeholder_initials">AB</string>
+    <string name="digit_four">4</string>
+    <string name="digit_five">5</string>
+
```

Comment:
remove the extra line

Reply:
Okay...will do

Reply:
@batbrain7 Changes were made...Please look after it